winit/src/conversion.rs

```diff
@@ -455,179 +449,125 @@ pub fn touch_event(
+            NamedKey::F16 => KeyCode::F16,
+            NamedKey::F17 => KeyCode::F17,
+            NamedKey::F18 => KeyCode::F18,
+            NamedKey::F19 => KeyCode::F19,
+            NamedKey::F20 => KeyCode::F20,
+            NamedKey::F21 => KeyCode::F21,
+            NamedKey::F22 => KeyCode::F22,
+            NamedKey::F23 => KeyCode::F23,
+            NamedKey::F24 => KeyCode::F24,
+            NamedKey::PrintScreen => KeyCode::Snapshot,
+            NamedKey::ScrollLock => KeyCode::Scroll,
+            NamedKey::Pause => KeyCode::Pause,
+            NamedKey::Insert => KeyCode::Insert,
+            NamedKey::Home => KeyCode::Home,
+            NamedKey::Delete => KeyCode::Delete,
+            NamedKey::End => KeyCode::End,
+            NamedKey::PageDown => KeyCode::PageDown,
+            NamedKey::PageUp => KeyCode::PageUp,
+            NamedKey::ArrowLeft => KeyCode::Left,
+            NamedKey::ArrowUp => KeyCode::Up,
+            NamedKey::ArrowRight => KeyCode::Right,
+            NamedKey::ArrowDown => KeyCode::Down,
+            NamedKey::Backspace => KeyCode::Backspace,
+            NamedKey::Enter => KeyCode::Enter,
+            NamedKey::Space => KeyCode::Space,
+            NamedKey::Compose => KeyCode::Compose,
+            NamedKey::NumLock => KeyCode::Numlock,
+            NamedKey::AppSwitch => KeyCode::Apps,
+            NamedKey::Convert => KeyCode::Convert,
+            NamedKey::LaunchMail => KeyCode::Mail,
+            NamedKey::MediaApps => KeyCode::MediaSelect,
+            NamedKey::MediaStop => KeyCode::MediaStop,
+            NamedKey::AudioVolumeMute => KeyCode::Mute,
+            NamedKey::MediaStepForward => KeyCode::NavigateForward,
+            NamedKey::MediaStepBackward => KeyCode::NavigateBackward,
+            NamedKey::MediaSkipForward => KeyCode::NextTrack,
+            NamedKey::NonConvert => KeyCode::NoConvert,
+            NamedKey::MediaPlayPause => KeyCode::PlayPause,
+            NamedKey::Power => KeyCode::Power,
+            NamedKey::MediaSkipBackward => KeyCode::PrevTrack,
+            NamedKey::PowerOff => KeyCode::Sleep,
+            NamedKey::Tab => KeyCode::Tab,
+            NamedKey::AudioVolumeDown => KeyCode::VolumeDown,
+            NamedKey::AudioVolumeUp => KeyCode::VolumeUp,
+            NamedKey::WakeUp => KeyCode::Wake,
+            NamedKey::BrowserBack => KeyCode::WebBack,
+            NamedKey::BrowserFavorites => KeyCode::WebFavorites,
+            NamedKey::BrowserForward => KeyCode::WebForward,
+            NamedKey::BrowserHome => KeyCode::WebHome,
+            NamedKey::BrowserRefresh => KeyCode::WebRefresh,
+            NamedKey::BrowserSearch => KeyCode::WebSearch,
+            NamedKey::BrowserStop => KeyCode::WebStop,
+            NamedKey::Copy => KeyCode::Copy,
+            NamedKey::Paste => KeyCode::Paste,
+            NamedKey::Cut => KeyCode::Cut,
+            _ => KeyCode::Unlabeled,
+        },
+        _ => KeyCode::Unlabeled,
```

Comment:
Aren't we missing some of the keys? Like `Minus`, `Plus`, `Equals`.
Those are just 3 I noticed are still part of `KeyCode` enum but are never constructed.

Reply:
Yep, still thinking about whether we should refactor further.